tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/documentation/ErrorsTest.java

```diff
@@ -52,10 +52,8 @@ public void setup() throws IOException {
 
     @Test(description = "Test error type")
     public void testErrorAsAType() {
-        Assert.assertEquals(testModule.errors.size(), 2, "Two errors expected");
+        Assert.assertEquals(testModule.errors.size(), 1, "One error expected");
         Assert.assertEquals(testModule.errors.get(0).name, "CacheError", "The error should be " +
                 "CacheError. But Found:" + testModule.errors.get(0).name);
-        Assert.assertEquals(testModule.errors.get(1).name, "Error", "The error should be " +
-                "Error. But Found:" + testModule.errors.get(1).name);
```

Comment:
why are we removing this assertion? was there any change to the bal file?

Reply:
Only one error is returned from the test now so had to fix the test in this manner

Reply:
This test will be fixed by the owner https://github.com/ballerina-platform/ballerina-lang/issues/27768